geonode_mapstore_client/templates/geonode-mapstore-client/app/dashboard.html

```diff
@@ -0,0 +1,5 @@
+{% load static %}
+{% load client_version %}
+
+{% include '../_geonode_config.html' with is_embed=is_embed is_app=True plugins_config_key=plugins_config_key|default:'dashboard' is_new_resource=is_new|default:'true' %}
+<script id="ms2-api" src="{% static 'mapstore/dist/gn-geostory.js' %}?{% client_version %}"></script>
```

Comment:
I think we can leave an empty page for this PR or at least remove the gn-geostory script because it used only by geostories applications. I think we will implement the dashboard app in a different PR